marlowe-runtime/src/Language/Marlowe/Runtime/Transaction/BuildConstraints.hs

```diff
@@ -43,4 +48,9 @@ buildWithdrawConstraints
   :: MarloweVersion v -- ^ The Marlowe version to build the transaction for.
   -> PayoutDatum v -- ^ The role token from which to withdraw funds.
   -> Either WithdrawError (TxConstraints v)
-buildWithdrawConstraints = error "not implemented"
+buildWithdrawConstraints =
+  \case MarloweV1 -> Right . buildWithdrawConstraintsV1
+  where
+    buildWithdrawConstraintsV1 :: AssetId -> TxConstraints 'V1
+    buildWithdrawConstraintsV1 assetId =
+      TxConstraints.mustConsumePayouts assetId <> TxConstraints.mustSpendRoleToken assetId
```

Comment:
We also need to add a `mustPayToAddress` here to send the funds to the `changeAddress`

Reply:
Ok, I'll fix it

Reply:
The `buildWithdrawConstraints` procedure doesn't have access to `Address`. I guess it was incorrectly designed?

Reply:
Disregard this comment - transaction balancing will handle this.